modules/virt-deprecated-tasks.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-deprecated-tasks_{context}"]
+= Removing deprecated or unused resources
+
+* The Tekton-tasks-operator (TTO) has been deprecated and removed in {VirtProductName} 4.17. You can clean up any remaining resources associated with the operator by running the following command.
+
+.Procedure
+
+. The following command removes any remaining tekton-related resources from the cluster:
+
+[source,terminal]
+----
+oc delete clusterroles,rolebindings,serviceaccounts,configmaps,pipelines,tasks \
+  --selector 'app.kubernetes.io/managed-by=ssp-operator' \
+  --selector 'app.kubernetes.io/component in (tektonPipelines,tektonTasks)' \
+  --selector 'app.kubernetes.io/name in (tekton-pipelines,tekton-tasks)' \
+  --ignore-not-found \
+  --all-namespaces
+----
+* If the Tekton tasks operator custom resource definitions (CRDs) have already been removed, the command may return an error. You can safely ignore this, as all other matching resources will still be deleted.
```

Comment:
```suggestion
* If the Tekton tasks operator custom resource definitions (CRDs) have already been removed, the command might return an error. You can safely ignore the error, as all other matching resources are still deleted.
```

Reply:
The use of may vs. might is covered in the IBM Style Guide [Word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m) section.

The use of tense is covered here in the [IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense).